### PR TITLE
Use react-styleguidist

### PR DESCRIPTION
**Pull request by roth1002**

### CODE REVIEW

File: styleguide.config.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const path = require('path');
+
+module.exports = {
+	title: 'React Radio',
+	webpackConfig: require('./webpack.config.js'),
+	components: ['examples/*Radio*.jsx', 'src/*.jsx'],
```
Comment (xJkit) [inline styleguide.config.js]: `examples/*.jsx` ?

Comment (roth1002) [reply styleguide.config.js]: Since no need Section.jsx/Navbar.jsx in exmaples folder

File: .gitignore [hunk 2]
```diff
@@ -2,6 +2,8 @@ node_modules
 npm-debug.log
 package-lock.json
 yarn.lock
+.DS_Store
```
Comment (leeabc) [inline .gitignore:5]: Please remove the useless file.

Comment (roth1002) [reply .gitignore:5]: WTF

Comment (leeabc) [reply .gitignore:5]: Sorry for misunderstanding. we should add this in gitignore file.

File: package.json [hunk 3]
```diff
@@ -8,16 +8,18 @@
     "lib"
   ],
   "scripts": {
-    "prepublish": "npm run eslint && npm test && npm run clean && npm run build && npm run build-examples",
+    "prepublish": "npm run eslint && npm test && npm run clean && npm run build && npm run styleguide:build && cp -a styleguide/* docs/",
```
Comment (leeabc) [inline package.json]: we can consider to create a symbolic link for docs then we don't need to cp & rm.

**Pull request merged**